gui/src/components/Sidebar.tsx

```diff
@@ -31,6 +33,7 @@ export const TABS = [
     component: Account,
     icon: RequestQuoteSharpIcon,
   },
+  { path: "nfts", name: "NFTs", component: Nfts, icon: ImageIcon },
```

Comment:
instead of a new sidebar item, I think NFTs should go under the Account page.

probably later with some better navigation inside it (when many items exist), but that's for later
keeping NFTs on a separate page when Account already handles native + ERC20 is very inconsistent

Reply:
About this, agree with the consistency but I don't agree that we should place everything under "Account" and mix erc20 and erc721 together, because it might become messy.
What about changing the "account" panel to "balances" or "tokens" and keep two separated pages ?

Reply:
I have no idea about where to put this. It makes sense to put it on the Account, but that would require some sort of navigation to change between ERC20 and ERC721, while at the same time, there's plenty of space on the sidebar for now.

Reply:
The simples approach I was thinking of this previously was to eventually have a submenu below "Account", which could just be made with anchor links within the same page. Meaning we can either scroll down and see the NFTs, or still navigate through the sidebar. for me this is the best of both worlds

the "Accounts" tab was already called "Balances" before, and the reason for changing that was precisely because I didn't want it to be just about ERC20s, but about the account as a whole